postgresql/main.fmf

```diff
@@ -0,0 +1,9 @@
+summary: Set of basic functions for postgresql
+contact:
+- Vaclav Danek <vdanek@redhat.com>
+test: ./runtest.sh
+framework: beakerlib
+recommend:
+- initscripts
+duration: 5m
+extra-task: /CoreOS/database/postgresql
```

Comment:
I would remove these two attributes.
Duration doesn't really make sense with library and extra-task is for downstream.

Reply:
Ok, will believe it doesn't make sense.